CWE/common/countries/Gran Colombia.txt

```diff
@@ -146,31 +135,31 @@ party = {
 }
 unit_names = {
 	carrier = {
-		"Almirante Padilla" Buenaventura "Cartagena de Indias" Colombia "Crist�bal Col�n" "Fco. de Paula Santander" "Gran Colombia" Independiente "Nueva Granada" "Sim�n Bol�var"
+		"Almirante Padilla" Buenaventura "Cartagena de Indias" Colombia "Cristóbal Colón" "Fco. de Paula Santander" "Gran Colombia" Independiente "Nueva Granada" "Simón Bolóvar"
```

Comment:
I believe the encoding has been changed from ANSI TO UTF in this file. This will break the special characters when it is merged.

Reply:
It probably is related to why this change is now present in every branch I try to make. I'll try to fix it, but I'm not exactly sure how it keeps putting itself back into place.

Reply:
It should be fixed now.